packages/story-editor/src/components/panels/shared/hotlinkModal.js

```diff
@@ -134,7 +135,12 @@ function HotlinkModal({
         needs_proxy: shouldProxy,
       });
 
-      onSelect({ src: link, needsProxy: shouldProxy });
+      onSelect({
+        ...hotlinkInfo,
+        id: uuidv4(),
```

Comment:
What's the ID for? 🤔 
I don't think we should start adding IDs to hotlinked media items unless there is a valid use case.

Reply:
Removed

Reply:
FIxed.